pycbc/results/pygrb_postprocessing_utils.py

```diff
@@ -377,14 +412,11 @@ def get_antenna_dist_factor(antenna, ra, dec, geocent_time, inc=0.0):
 # =============================================================================
 # Function to calculate the detection statistic of a list of triggers
 # =============================================================================
-def get_bestnrs(trigs, q=4.0, n=3.0, null_thresh=(4.25, 6), snr_threshold=6.,
-                sngl_snr_threshold=4., chisq_threshold=None,
-                null_grad_thresh=20., null_grad_val=0.2):
+def get_bestnrs(trigs, ifos, snr_threshold=6., sngl_snr_threshold=4.,
```

Comment:
As far as I know, we can likely get rid of this function since we're reweighting the SNR in `pycbc_multi_inspiral`